app/models/employment/Benefits.scala

```diff
@@ -103,12 +119,11 @@ case class Benefits(accommodation: Option[BigDecimal] = None,
   def toBenefitsViewModel(isUsingCustomerData: Boolean, submittedOn: Option[String] = None,
                           cyaBenefits: Option[BenefitsViewModel] = None): BenefitsViewModel = {
     BenefitsViewModel(
-      carVanFuelModel = carVanFuelSection(cyaBenefits),
-      accommodation, assets, assetTransfer, beneficialLoan, educationalServices, entertaining, expenses, medicalInsurance,
-      telephone, service, taxableExpenses,  nonQualifyingRelocationExpenses, nurseryPlaces, otherItems,
-      paymentsOnEmployeesBehalf, personalIncidentalExpenses, qualifyingRelocationExpenses, employerProvidedProfessionalSubscriptions,
+      accommodationRelocationModel = accommodationRelocationSection(cyaBenefits),
```

Comment:
same here - can you move the accommodationRelocationModel so it's after the carVanFuelModel